global_helpers/panther_iocs.py

```diff
@@ -34,7 +34,663 @@
     "d0d626deb3f9484e649294a8dfa814c5568f846d5aa02d4cdad5d041a29d5600",
 }
 
+#LOG4J IOCs:
+# IPs Pulled from Greynoise and compiled here:
+# https://gist.github.com/gnremy/c546c7911d5f876f263309d7161a7217
+# Created 12-13-21
 
+LOG4J_IP_IOCS = {
```

Comment:
would suggest adding a comment of where you got it from? Here's some others

https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Sample%20Data/Feeds/Log4j_IOC_List.csv
https://github.com/Malwar3Ninja/Exploitation-of-Log4j2-CVE-2021-44228/blob/main/Threatview.io-log4j2-IOC-list

Reply:
https://github.com/panther-labs/panther-analysis/pull/356/files#diff-56771479b9258fa730e354c5959c7c1228ef8a7d5863a78db8f48e0f48ef93b5R38

Reply:
Added and deduped